packages/common/src/utils/object/common.ts

```diff
@@ -89,3 +89,6 @@ export const omitBy: PickOmitBy = (obj: AnyRecord, predicate: (key: string, valu
  * @deprecated use pickBy instead
  */
 export const filterEntries = pickBy;
+
+export const map = <T, R>(obj: Record<string | number | symbol, T>, callback: (value: T) => R) =>
```

Comment:
should probably be named `mapValue` so we can differentiate it if we add a `mapKey` in the future